clifford/dcga.py

```diff
@@ -0,0 +1,123 @@
+"""
+The Cl(8,2) Double Conformal Geometric Algebra
```

Comment:
Should this file be `dg3c`, since it's the 3d one?

Reply:
Or `g3dc`

Reply:
I don't know what the best thing to do here is.. mostly because I don't really like g3c as a name but nonetheless we already have it. I think maybe dg3c is the way to go